app/models/measure.rb

```diff
@@ -179,6 +179,12 @@ def measure_conditions_with_guidance
     measure_conditions.select(&:has_guidance?)
   end
 
+  def measure_type_duty_label
+    return unless geographical_area.id && measure_type.id
+
+    I18n.t("measure_type_duties.#{geographical_area.id}.#{measure_type.id}", default: nil)
```

Comment:
```suggestion
    I18n.t("measure_type_duties.#{geographical_area&.id}.#{measure_type&.id}", default: nil)
```

You can drop the guard above with this syntax. I honestly don't think it's necessary in real life, though. Possibly you added it for the tests